pr_agent/servers/bitbucket_server_webhook.py

```diff
@@ -13,35 +14,55 @@
 from pr_agent.agent.pr_agent import PRAgent
 from pr_agent.config_loader import get_settings
 from pr_agent.log import get_logger
+from pr_agent.servers.utils import verify_signature
 
 router = APIRouter()
 
 
-def handle_request(background_tasks: BackgroundTasks, url: str, body: str, log_context: dict):
+def handle_request(
+    background_tasks: BackgroundTasks, url: str, body: str, log_context: dict
+):
     log_context["action"] = body
-    log_context["event"] = "pull_request" if body == "review" else "comment"
     log_context["api_url"] = url
     with get_logger().contextualize(**log_context):
         background_tasks.add_task(PRAgent().handle_request, url, body)
 
 
-@router.post("/webhook")
+@router.post("/")
 async def handle_webhook(background_tasks: BackgroundTasks, request: Request):
     log_context = {"server_type": "bitbucket_server"}
     data = await request.json()
     get_logger().info(json.dumps(data))
 
-    pr_id = data['pullRequest']['id']
-    repository_name = data['pullRequest']['toRef']['repository']['slug']
-    project_name = data['pullRequest']['toRef']['repository']['project']['key']
+    webhook_secret = getattr(get_settings().bitbucket_server, "webhook_secret", None)
```

Comment:
You can use get_settings().get("BITBUCKET_SERVER.WEBHOOK_SECRET") for consistency with this repository